css/home-page.styl

```diff
@@ -567,6 +580,7 @@ $flex-section
       flex-direction: column
       margin-right: 3em
       max-width: 100%
+      padding-bottom: 5em
```

Comment:
The button was too close to the next block: 

<img width="443" alt="screen shot 2019-01-23 at 10 29 56 pm" src="https://user-images.githubusercontent.com/3202211/51653873-5fbdac00-1f63-11e9-80a1-9ab04b23a08b.png">

I set it to 5em to match the spacing of the spacing between other blocks on the page: 

<img width="389" alt="screen shot 2019-01-23 at 11 05 46 pm" src="https://user-images.githubusercontent.com/3202211/51653914-85e34c00-1f63-11e9-9c9c-443a260e2c45.png">